src/components/ProductForm/ProductForm.tsx

```diff
@@ -64,90 +69,34 @@ export const ProductForm = (): ReactElement => {
             <FontAwesomeIcon icon={faPen} />
           </div>
           <div className="col-span-2">
-            <label className={cn(subHeadingClass)} htmlFor="product_create_form_name">
-              Item Name
-            </label>
-            <input
-              className="w-full border-b-2 font-[600] focus:border-black focus:outline-none"
-              id="product_create_form_name"
-              onChange={(event) => setFormProduct({ ...formProduct, name: event.target.value })}
-              type="text"
-              value={formProduct.name}
-              data-testid="product-name"
-            />
+            <Input type="text" label="Item name" id="item-name" error={errors.name?.message} {...register('name')} />
           </div>
           <div className="col-span-2 row-span-2">
-            <label className={cn(subHeadingClass)} htmlFor="product_create_form_desc">
-              Item Description
-            </label>
-            <textarea
-              className="w-full resize-none border-b-2 font-[600] focus:border-black focus:outline-none"
-              id="product_create_form_desc"
-              onChange={(event) => setFormProduct({ ...formProduct, desc: event.target.value })}
-              rows={3}
-              value={formProduct.desc}
-              data-testid="product-description"
-            />
+            <TextArea label="Item description" id="item-description" error={errors.desc?.message} {...register('desc')} />
           </div>
           <div className="col-span-2">
-            <label className={cn(subHeadingClass, 'w-full')} htmlFor="product_create_form_category">
-              Category
-            </label>
-            <Dropdown options={categories} className="w-full" id="product_create_form_category" />
+            <Select label="Category" options={categoryOptions} id="category" error={errors.category?.message} {...register('category')} />
```

Comment:
When using the form to create a new product there should be a `defaultValue` for the category.
![details - Copy](https://user-images.githubusercontent.com/116075166/219372931-d8ad77b7-f31b-4772-9000-2eff5b641039.png)
It should either default to a placeholder like "Select category" (or similar) or have the first category as a default value on line 44.

Reply:
should be fixed now